File: Python3/프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    answer = ''
    participant.sort()
    completion.sort()
    for i in range(len(completion)):
        if participant[i] != completion[i]:
            answer = str(participant[i])
            break;
    if answer == '':
        answer = str(participant[len(participant)-1])
    return answer

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.js
function solution(participant, completion) {
    var answer = '';
    let i;
    // 참가자, 완주자 정렬
    participant = participant.sort();
    completion = completion.sort()
    
    // 비교해서 없는애 추출
    for(i = 0; i < completion.length; i++){
        if(participant[i] !== completion[i]){
            return participant[i];
        }
    }
    
    return participant[i];
}